docs/intro/tutorial01.txt

```diff
@@ -206,8 +206,8 @@ rather than creating directories.
 
 .. admonition:: Projects vs. apps
 
-    What's the difference between a project and an app? An app is a Web
-    application that does something -- e.g., a Weblog system, a database of
+    What's the difference between a project and an app? An app is a web
+    application that does something -- e.g., a weblog system, a database of
```

Comment:
No one seems to use the `weblog` term anymore: 
```suggestion
    application that does something -- e.g., a blog system, a database of
```

Reply:
An online store / shopping website might be a better example today since not all blogs "do something" (since rise of static site generators).

Reply:
In this context I think an online store / shopping website be a project? 🤔

Reply:
Let's focus this PR only on spelling changes. We can discuss this separately.